benchmarks/dynamo/ci_expected_accuracy/inference_huggingface.csv

```diff
@@ -0,0 +1,43 @@
+dev,name,batch_size,accuracy,calls_captured,unique_graphs,graph_breaks,unique_graph_breaks
+cuda,AlbertForMaskedLM,1,pass,439,1,0,0
```

Comment:
Something I did not realize until seeing this, is that we get a csv per shard. This means that I have to click through and download *eight* times to get all of them. Maybe we should have a script that downloads all the csvs from a job...

Reply:
yea, i am wanting to build more tooling for this too. i have some other ideas...